charger/config.go

```diff
@@ -27,6 +27,8 @@ func NewFromConfig(log *api.Logger, typ string, other map[string]interface{}) ap
 		c = NewEVSEWifiFromConfig(log, other)
 	case "simpleevse", "evse":
 		c = NewSimpleEVSEFromConfig(log, other)
+	case "porsche", "audi", "bentley", "mcc":
+		c = NewMobileConnectFromConfig(log, other)
```

Comment:
Wäre nicht evtl mcc oder mobileconnect hier besser? Not sure :/

Reply:
Bin ich recht leidenschaftslos. Sag du :)